Makefile

```diff
@@ -36,7 +36,7 @@ docs:
 	$(MAKE) -C docs html
 
 release: clean
-	python setup.py sdist bdist_wheel upload
```

Comment:
It came with the `Makefile` template I used. It doesn't work, so I removed it for now. We can always add it back later if for some reason it was actually useful.